Sources/BFKit/iOS/UIKit/UIViewControllerExtension.swift

```diff
@@ -55,4 +55,16 @@ extension UIViewController {
             }
         }
     }
+    
+    
+    /// Presents a UIAlertController with a title, message and a button to dismiss the controller.
+    ///
+    /// - parameter title: The title of the UIAlerController.
+    /// - parameter message: An optional String for the UIAlertController's message.
+    /// - parameter buttonTitle: The title of the dismiss button: Defaults to "OK".
+    public func userErrorAlert(title: String, message: String?, buttonTitle: String = "OK") {
```

Comment:
I don't really like this functon, we can make it more generic like `present(title: String, message: String, actions: [UIAlertAction], alertType: UIAlertControllerStyle = .alert)`